crates/precompile/src/kzg_point_evaluation.rs

```diff
@@ -86,7 +96,122 @@ pub fn verify_kzg_proof(commitment: &Bytes48, z: &Bytes32, y: &Bytes32, proof: &
             let env = kzg_rs::EnvKzgSettings::default();
             let kzg_settings = env.get();
             KzgProof::verify_kzg_proof(commitment, z, y, proof, kzg_settings).unwrap_or(false)
+        } else {
+            bls12_381_backend::verify_kzg_proof(commitment, z, y, proof)
```

Comment:
Avoids a KzgSettings file given that revm never needs to create proofs and the setup information needed to verify proofs is fixed and small